src/base-element.js

```diff
@@ -491,6 +498,14 @@ export class BaseElement {
   }
 
   /**
+   * Minimum event trust required for activate().
+   * @return {ActionTrust}
+   */
+  activationTrust() {
```

Comment:
Technically not yet, but only because the default actions for all extensions *happen to* require `ActionTrust.MEDIUM`. We should allow extensions to customize this though.